jsk_pcl_ros_utils/src/plane_concatenator_nodelet.cpp

```diff
@@ -219,7 +219,9 @@ namespace jsk_pcl_ros_utils
     pcl::ModelCoefficients::Ptr refined_coefficients(new pcl::ModelCoefficients);
     seg.segment(*refined_inliers, *refined_coefficients);
     if (refined_inliers->indices.size() > 0) {
-      return refined_coefficients;
+      pcl::ModelCoefficients::Ptr fixed_refined_coefficients(new pcl::ModelCoefficients);
+      forceToDirectOrigin(refined_coefficients, fixed_refined_coefficients);
+      return fixed_refined_coefficients;
```

Comment:
Is it enough to change the orientation of fixed_refined_coefficients without changing the orientation of the original_coefficients?

Reply:
I think it depends on the nodelet design.
One thing I noticed is that the RANSAC estimation has a different direction of the coefficients depending on the angle of the plane, so I only do it if the plane is refined by RANSAC.

If the output should be aligned to point close to the origin for any input, then the original_coefficients should also change direction, but in that case, I think we need to make sure that the order of the points in the polygon is also reversed.
On the other hand, I don't think there is any need to change the original_coefficients if the input is assumed to have the same orientation of the coefficients as the orientation of the vertical vector derived from the order of the polygon points.

Reply:
Thank you for your explanation.
Could you add the explanation for the direction of coefficients in the following document?
https://github.com/jsk-ros-pkg/jsk_recognition/blob/master/doc/jsk_pcl_ros_utils/nodes/plane_concatenator.md

Reply:
Thank you!